framework/include/postprocessors/GreaterThanLessThanPostprocessor.h

```diff
@@ -0,0 +1,56 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralPostprocessor.h"
+
+// Forward Declarations
+class GreaterThanLessThanPostprocessor;
+class MooseVariableFEBase;
+namespace libMesh
+{
+class MeshBase;
+}
+
+template <>
+InputParameters validParams<GreaterThanLessThanPostprocessor>();
+
+class GreaterThanLessThanPostprocessor : public GeneralPostprocessor
+{
+public:
+  GreaterThanLessThanPostprocessor(const InputParameters & parameters);
+
+  void initialize() override;
+  void execute() override;
+
+  PostprocessorValue getValue() override;
+
+private:
+  /// MOOSE variable we compute the contact set from
+  const MooseVariableFEBase & _var;
+
+  /// The libmesh mesh
+  const MeshBase & _mesh;
+
```

Comment:
Should store `MooseMesh`, and `MeshBase` can be retrieved from `MooseMesh`

Reply:
But the object I really want to call methods on is the `MeshBase`. If I don't call `MooseMesh::getMesh` once in the constructor, then I'll be calling it every time the `GreaterThanLessThanPostprocessor::execute` method gets called.